core/src/main/scala/org/apache/spark/h2o/utils/H2OContextUtils.scala

```diff
@@ -77,6 +84,103 @@ private[spark] trait H2OContextUtils extends Logging {
     }
   }
 
+  /**
+    * @param destination directory where the logs will be downloaded
+    */
+  def downloadH2OLogs(destination: URI): URI = {
+    val perNodeZipByteArray = H2O.CLOUD.members.zipWithIndex.map { case (node, i) =>
+      // Skip nodes that aren't healthy, since they are likely to cause the entire process to hang.
+      try {
+        if (node.isHealthy) {
+          val g = new GetLogsFromNode
```

Comment:
interesting i would expect `new GetLogsFromNode(i).doIt().bytes()`

Reply:
Me too, but current H2O API does not allow that  and I wanted to limit the scope of this change just to SW